standard/standard_8.php

```diff
@@ -143,7 +143,7 @@ function metaphone(string $string, int $max_phonemes = 0) {}
 /**
  * Turn on output buffering
  * @link https://php.net/manual/en/function.ob-start.php
- * @param callable $callback [optional] <p>
+ * @param callable|null $callback [optional] <p>
```

Comment:
@VincentLanglet since you have added `null` as a default value in signature, `[optional]` can be removed here. It's clear that parameter is optional based on signature